README.md

```diff
@@ -85,7 +85,7 @@ Add `https://{some-funky-words}.edgecompute.app/callback` to the list of allowed
 
 ## The flow in detail
 
-![Edge authentication flow diagram](https://user-images.githubusercontent.com/12828487/111877650-1c70f380-899c-11eb-98ba-427e4006f58a.png)
+![Edge authentication flow diagram](https://user-images.githubusercontent.com/12828487/115379253-4438be80-a1c9-11eb-81af-9470e324434a.png)
```

Comment:
Should this be in the repo?

Reply:
I'd rather exclude it from the repo, because it makes using it as a template easier.